src/code42cli/cmds/auditlogs.py

```diff
@@ -0,0 +1,159 @@
+import click
+
+from code42cli.cmds.search.extraction import create_simple_send_to_handler
+from code42cli.date_helper import parse_max_timestamp
+from code42cli.date_helper import parse_min_timestamp
+from code42cli.logger import get_logger_for_server
+from code42cli.options import OrderedGroup
+from code42cli.options import sdk_options
+from code42cli.options import server_options
+
+EVENT_KEY = "events"
+
+
+filter_option_usernames = click.option(
+    "--usernames", required=False, help="Filter results by usernames.", multiple=True,
+)
+filter_option_user_ids = click.option(
+    "--user-ids", required=False, help="Filter results by user ids.", multiple=True,
+)
+
+filter_option_user_ip_addresses = click.option(
+    "--user-ips",
+    required=False,
+    help="Filter results by user ip addresses.",
+    multiple=True,
+)
+filter_option_affected_user_ids = click.option(
+    "--affected-user-ids",
+    required=False,
+    help="Filter results by affected user ids.",
+    multiple=True,
+)
+filter_option_affected_usernames = click.option(
+    "--affected-usernames",
+    required=False,
+    help="Filter results by affected usernames.",
+    multiple=True,
+)
+filter_option_event_types = click.option(
+    "--event-types",
+    required=False,
+    help="Filter results by event types.",
+    multiple=True,
+)
+
+filter_option_begin_time = click.option(
```

Comment:
i think that begin / end time arguments exist in another place in the codebase, can we import them and share those definitions instead of copying them over here? Are the time formats for audit log the same as what is used for file events / alerts?

Reply:
Refactored, there are subtle differences in usage so had skipped it in the first iteration. Thanks.